### ISSUE TITLE
SCTE-35 Data Lost

### ISSUE THREAD

**Issue by vsawma** [futzu/six2scte35#1]
We are using the six2scte35 tool as follows:

ffmpeg -f mpegts -i <some live stream> -map 0:d -codec:d copy -map 0:v -codec:v h264 -map 0:a -codec:a aac -f mpegts - | six2scte35 -p 0x3e8 -o out.ts

The scte35 stream gets renamed properly.
Yet, the data itself within the stream gets lost.

  Stream #0:2[0x3e8]: Data: scte_35

What are we missing here?

**Comment by futzu** [comment]
Can you send me a link to the video, or just a piece of it before you processed it with ffmpeg?

**Comment by futzu** [comment]
**Ffmpeg changed rthe pids of the video.**

here was my input 
```rebol
 Duration: 00:16:21.01, start: 2024.020000, bitrate: 680 kb/s
  Program 1 
  Stream #0:0[0x1e5]: Data: scte_35
  Stream #0:1[0x1e1]: Video: h264 (Main) ([27][0][0][0] / 0x001B), yuv420p(progressive), 640x360 [SAR 1:1 DAR 16:9], Closed Captions, 29.97 fps, 29.97 tbr, 90k tbn
  Stream #0:2[0x1e2](eng): Audio: aac (LC) ([15][0][0][0] / 0x000F), 44100 Hz, stereo, fltp, 96 kb/s
  Stream #0:3[0x1e3]: Data: scte_35
  Stream #0:4[0x1e4]: Data: timed_id3 (ID3  / 0x20334449)
```
I ran the same command you had 
```lua
ffmpeg  -i sk.ts  -map 0:d -codec:d copy -map 0:v -codec:v h264 -map 0:a -codec:a aac skout.ts
```
FYI make sure to add `-copyts` so ffmpeg doesn't change  the PTS.

my output looks like 
```lua

  Program 1 
    Metadata:
      service_name    : Service01
      service_provider: FFmpeg
  Stream #0:0[0x100]: Data: bin_data ([6][0][0][0] / 0x0006)
  Stream #0:1[0x101]: Data: bin_data ([6][0][0][0] / 0x0006)
  Stream #0:2[0x102]: Data: timed_id3 (ID3  / 0x20334449)
  Stream #0:3[0x103]: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p(progressive), 640x360 [SAR 1:1 DAR 16:9], Closed Captions, 29.97 fps, 29.97 tbr, 90k tbn
  Stream #0:4[0x104](eng): Audio: aac (LC) ([15][0][0][0] / 0x000F), 44100 Hz, stereo, fltp, 137 kb/s
```


You can use  `-mpegts_start_pid`
